tests/config.c

```diff
@@ -115,6 +119,12 @@ config_usage()
    platform_error_log("\t--cache-capacity-mib (%d)\n",
                       (int)(TEST_CONFIG_DEFAULT_CACHE_SIZE_GB * KiB));
    platform_error_log("\t--cache-debug-log\n");
+   platform_error_log("\t--perform-bg-tasks\n");
+   platform_error_log("\t--no-perform-bg-tasks\n");
+   platform_error_log("\t--num-background-threads (%d)\n",
```

Comment:
I think these 4 lines can come out as these are not the supported args: `--num-background-threads` and `--num-background-memtable-threads`.

The ones I finalized in a previous check-in are on lines 137 & 140: `--num-normal-bg-threads` and `--num-memtable-bg-threads`.

The choice of these `--args` is to keep their names consistent with the fields in `master_config{}` that store these config params.

Reply:
Agreed, except that --perform-bg-tasks controls whether _foreground_ threads perform bg tasks.  It is in use.